tutorials/tutorial-01.qmd

```diff
@@ -6,33 +6,1403 @@ time: "~45 minutes"
+| `natura2000areas` | Natura 2000 network, the EU-wide network of protected areas established under the Birds and Habitats Directives to protect Europe's most valuable species and habitats |
+| `marineprotectedareas` | Other marine protected areas, sites designated under regional sea conventions (OSPAR, HELCOM, Barcelona Convention) and other international frameworks |
+
+**Energy Infrastructure:**
+
+| Layer | Description |
+|-------|-------------|
+| `platforms` | Offshore platforms, including oil, gas, and wind energy installations such as production platforms, substations, and support structures |
+| `pipelines` | Subsea pipelines for oil, gas, and chemical transport on the seabed |
+| `pcablesbshcontis` | Power cables from BSH (German Federal Maritime and Hydrographic Agency) and CONTIS database |
+| `pcablesshom` | Power cables from SHOM (French Naval Hydrographic and Oceanographic Service) |
+| `pcablesrijks` | Power cables from Rijkswaterstaat (Dutch Directorate-General for Public Works and Water Management) |
+| `pcablesnve` | Power cables from Norwegian sources |
+
+Note: EMODnet also provides telecommunication cable datasets (e.g., `sigcables`, `ukfibrecables`), but we focus on energy infrastructure in this tutorial.
+
+## Setup
+
+### Packages
+
+We'll use `emodnet.wfs` to access EMODnet WFS services, `sf` for spatial data handling, `dplyr` and `purrr` for data manipulation, and `ggplot2` and `tmap` for visualisation.
+
+```{r}
+#| label: load-packages
+#| message: false
+library(emodnet.wfs)
+library(sf)
+library(dplyr)
+library(purrr)
+library(ggplot2)
+library(tmap)
+```
+
+```{r}
+#| label: setup
+#| include: false
+# Load package data (mpa_attr_tbls, infra_attr_tbls) and helper functions
+library(emodnet.bio.r.geo.tutorials)
+```
+
+::: {.callout-tip collapse="true"}
+## Package installation
+If you haven't installed these packages, run:
+
+```r
+install.packages(c("emodnet.wfs", "sf", "dplyr", "purrr", "ggplot2", "tmap"))
+```
+
+For the latest development version:
+
+```r
+# install.packages("pak")
+pak::pak("EMODnet/emodnet.wfs")
+```
+:::
+
+### Study Area
+
+We'll focus on the North Sea, defined by the following [bounding box](../concepts.qmd#bounding-boxes-and-spatial-queries):
+
+```{r}
+#| label: define-study-area-bbox
+# Geographic bbox for WFS queries (EPSG:4326)
+north_sea_bbox <- c(xmin = -4, ymin = 51, xmax = 9, ymax = 62)
+
+# Create bbox polygon and transform to ETRS89-LAEA (EPSG:3035) for analysis
+bbox_polygon <- sf::st_as_sfc(sf::st_bbox(north_sea_bbox, crs = 4326))
+bbox_3035 <- sf::st_transform(bbox_polygon, 3035)
+```
+
+We'll use [EPSG:3035](../concepts.qmd#coordinate-reference-systems-crs) (ETRS89-LAEA) as our working CRS, a projected coordinate system with metric units that's well-suited for European marine analysis.
+
+```{r}
+#| label: fig-study-area
+#| echo: false
```

Comment:
I'd actually be in favor of `echo: true`, as collapsed code maybe?

Reply:
Good idea, changed to `code-fold: true`.